.travis.yml

```diff
@@ -2,14 +2,17 @@
 sudo: false
 language: scala
 scala:
-  - 2.11.12
   - 2.12.10
   - 2.13.1
 
 env:
   - TRAVIS_JDK=8
   - TRAVIS_JDK=11
 
+jobs:
+  allow_failures:
+    - env: TRAVIS_JDK=11
```

Comment:
@jamesward building on Java 11 but not failing the build for now. We can remove this later when the build is green for this version of JDK.